lib/models/package-info-cache/index.js

```diff
@@ -561,7 +478,7 @@ class PackageInfoCache {
       if (!pathFailed) {
         // we have a valid realPath
         let packageJsonPath = path.join(realPath, PACKAGE_JSON);
-        let pkgfile = getRealFilePath(packageJsonPath);
+        let pkgfile = resolvePackageCache(realPath);
```

Comment:
Given what you have at lines 17 & 18, shouldn't this remain as 'getRealFilePath'?

Reply:
yes, oops moving to quick.

Reply:
whoaaaaaa!!! :-)